espresso-server/app/src/androidTest/java/io/appium/espressoserver/Http/Server.java

```diff
@@ -13,7 +13,13 @@
 import static android.support.test.espresso.action.ViewActions.click;
 import static android.support.test.espresso.matcher.ViewMatchers.withText;
 
+import android.app.Instrumentation;
+import android.app.Instrumentation.ActivityMonitor;
+import android.content.Intent;
+import android.support.test.InstrumentationRegistry;
+
 public class Server extends NanoHTTPD {
+    private boolean opened = false;
```

Comment:
This is purely temporary, so that the initial test can pass. The actual logic in here will be entirely different.